src/utils/dataset.py

```diff
@@ -74,3 +76,8 @@ def register_detectron2_datasets(name, images_dir, splits, min_bbox_area=200, li
                                                                  lim))
 
         MetadataCatalog.get(f"{name}_{d}").set(thing_classes=["SickTrees"])
+
+def image_reader(file_name, format=None):
```

Comment:
to raczej niepotrzebne
1) argument format jest nieużywany
2) handler f jest nieużywany (nie ma sensu otwieranie pliku po nazwie, tylko po to, żeby z handlera wydobyć nazwę
3) można to zastąpić po prostu: `cv2.imread(f.name, cv2.IMREAD_UNCHANGED)`